### ISSUE TITLE
Optional delay

### ISSUE THREAD

**Issue by elazar55** [MelanX/ToolSwap#32]
Is it possible to add an optional parameter for a delay before swapping tools? This is specifically to avoid triggering anti-cheat plug-ins on servers.
Thank you.

**Comment by MelanX** [comment]
I already tried to fix things which triggered anti-cheat plug-ins which didn't work. One of my tries was to add an delay which didn't help, even when setting it to 5 seconds.
For that, I worked with two server owners together and they also asked the anti-cheat plug-in dev for some code for this specific issue. That didn't help, we all don't understand what's wrong here.

I could add delay if you still want, but it doesn't help to avoid these plug-ins. At least the plug-in `Vulcan` I guess.
